hw/ip/otbn/dv/otbnsim/sim/csr.py

```diff
@@ -52,13 +65,6 @@ def read_unsigned(self, wsrs: WSRFile, idx: int) -> int:
     def write_unsigned(self, wsrs: WSRFile, idx: int, value: int) -> None:
         assert 0 <= value < (1 << 32)
 
-        # The RISC-V Privileged Specification v1.11 describes an address map
-        # for custom CSRs which depends on them being read-only or read-write.
-        if not (idx >= 0x7C0 and idx <= 0x7FF):
-            raise RuntimeError('CSR {:#x} not in writable range (0x7c0 - '
-                               '0x7ff), according to the RISC-V Privileged '
-                               'Specification v1.11.'.format(idx))
-
```

Comment:
I added this part to ensure that we don't add more CSRs which don't follow this address range rule, so I'd prefer to leave it in somewhere.